modules/jt-examples-of-common-use-cases.adoc

```diff
@@ -0,0 +1,169 @@
+:_content-type: CONCEPT
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-examples-of-common-use-cases_{context}"]
+= Examples of common use cases
+
+Both Jenkins and Tekton offer capabilities for common CI/CD use cases, such as: 
+
+* Compiling, building images, and deploying images
+* Extending the core capabilities by using plugins
+* Reusing shareable libraries and custom scripts
+
+This section briefly describes the comparable workflows for some common use cases in Jenkins and Tekton.
```

Comment:
IBM SG page 22: avoid self referential language, like `This section describes...` 

I think this entire sentence can be omitted. I think seeing the examples of use cases sets the users up to continue reading documentation to learn more about the use cases and capabilities.